webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -2,6 +2,8 @@
 /* eslint-disable import/no-dynamic-require */
 import Jed from 'jed';
 import { addLocaleData } from 'react-intl';
+import Cookies from 'js-cookie';
+import jstz from 'jstz';
```

Comment:
Where and how did this previously get imported?

Reply:
I am not sure, so this seems like a better place for it. It somehow exsisted in app/assets/javascripts/application.js and its installed in foreman js